app/models/job.rb

```diff
@@ -3,4 +3,7 @@ class Job < ActiveRecord::Base
   STATUSES = [:draft, :published, :archived]
   enumerize :state, in: STATUSES
   belongs_to :organization
+  belongs_to :published_by, foreign_key: :user_id, class_name: 'User'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.